include/seccomp.h.in

```diff
@@ -198,44 +198,108 @@ struct scmp_arg_cmp {
  */
 #define SCMP_SYS(x)		(__NR_##x)
 
+/* Helper macros for the 32-bit argument comparison macros */
```

Comment:
I would suggest adding a warning that these macros should not be used directly, for example:

    /* Helpers for the argument comparison macros, DO NOT USE directly */